addons/pvr.mediaportal.tvserver/src/Socket.cpp

```diff
@@ -138,7 +138,7 @@ bool Socket::bind ( const unsigned short port )
   _sockaddr.sin_addr.s_addr = INADDR_ANY;  //listen to all
   _sockaddr.sin_port = htons( port );
 
-  int bind_return = ::bind(_sd, (sockaddr*)(&_sockaddr), sizeof(_sockaddr));
+  int bind_return = MPTV::bind(_sd, reinterpret_cast<sockaddr*>(&_sockaddr), sizeof(_sockaddr));
```

Comment:
find . -name "*.h" | xargs grep "using namespace std;" <- how did THAT pass your shit filter!